site2/docs/io-cli.md

```diff
@@ -557,7 +557,9 @@ $ pulsar-admin sinks localrun options
 | `--client-auth-plugin` | Client authentication plugin using which function-process can connect to broker.
 |`--cpu`|The CPU (in cores) that needs to be allocated per sink instance (applicable only to the Docker runtime).
 | `--custom-schema-inputs` | The map of input topics to Schema types or class names (as a JSON string).
-| `--custom-serde-inputs` | The map of input topics to SerDe class names (as a JSON string).
+| `--max-redeliver-count` | Maximum number of times that a message is redelivered before being sent to the dead letter queue.
+| --dead-letter-topic | Name of the dead letter topic where the failing messages are sent.
+|| `--custom-serde-inputs` | The map of input topics to SerDe class names (as a JSON string).
```

Comment:
`||` => `|`

Reply:
Thanks for your comments, the doc is updated.